### PR TITLE
Adding Mikel as an approver

### PR DESCRIPTION
**Pull request by reyang**
Mikel has been actively contributing to OpenTelemetry .NET SDK across many modules, including the Jaeger and Zipkin exporters, integration with AspNet/Sql, as well as general improvements to the CI and infrastructure. He is also actively driving the better design of OpenTelemetry .NET API/SDK.

These are all the PRs from Mikel: https://github.com/open-telemetry/opentelemetry-dotnet/pulls?q=is%3Apr+author%3ACodeBlanch.

@CodeBlanch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Approvers ([@open-telemetry/dotnet-approvers](https://github.com/orgs/open-telem
 - [Bruno Garcia](https://github.com/bruno-garcia), Sentry
 - [Christoph Neumueller](https://github.com/discostu105), Dynatrace
 - [Liudmila Molkova](https://github.com/lmolkova), Microsoft
+- [Mikel Blanchard](https://github.com/CodeBlanch)
```
Comment (reyang) [inline README.md:49]: This has been discussed during the 6/2, 2020 SIG meeting.

@CodeBlanch would you confirm which company shall we put here?

Comment (CodeBlanch) [reply README.md:49]: @reyang Please put CoStar Group on there. Sorry for the delay, I wanted to get approval from my boss.

Comment (reyang) [reply README.md:49]: Sure. And no worry - we're not going to merge this PR until we get your explicit approval 😄.
Thanks for all your contributions btw!

**Comment by MikeGoldsmith** [comment]
@codeblanch you happy for this to be merged?

**Comment by CodeBlanch** [comment]
@MikeGoldsmith Ya go for it. Some trepidation but I will do my best to help out :)

**Comment by SergeyKanzhelev** [comment]
added @CodeBlanch to the approvers team

**Pull request merged**